src/main/java/net/minecraftforge/fml/common/ProxyInjector.java

```diff
@@ -42,7 +45,12 @@ public static void inject(ModContainer mod, ASMDataTable data, Side side, ILangu
     {
         FMLLog.log.debug("Attempting to inject @SidedProxy classes into {}", mod.getModId());
         SetMultimap<String, ASMData> modData = data.getAnnotationsFor(mod);
-        Set<ASMData> mods = modData.get(Mod.class.getName());
+        Set<ASMDataTable.ASMData> mods = Sets.newHashSet();
```

Comment:
Why alter this declaration?

Reply:
ah, probably because he uses idea's default formatter which does not insert inner class imports.